src/main/java/org/prebid/server/bidder/colossus/ColossusBidder.java

```diff
@@ -131,10 +131,13 @@ private static BidderBid makeBidderBid(Bid bid, List<Imp> imps, String currency,
     private static BidType resolveBidType(String impId, List<Imp> imps) {
```

Comment:
As you can see [here](https://github.com/prebid/prebid-server/pull/2205/files#diff-33a4547feab44fa7b7a84a795b4094605bfb2728c8dd8dfa2ab9bac5b0acc43cR124-R136) firstly you need to check `Bid.Ext` that may contain `BidType`.